apps/src/templates/courseOverview/TeacherCourseOverview.tsx

```diff
@@ -14,10 +14,9 @@ import {
   setVerifiedResources,
 } from '@cdo/apps/code-studio/verifiedInstructorRedux';
 import {setViewType, ViewType} from '@cdo/apps/code-studio/viewAsRedux';
-import {getStore} from '@cdo/apps/redux';
 import {NotificationType} from '@cdo/apps/sharedComponents/Notification';
 import Spinner from '@cdo/apps/sharedComponents/Spinner';
-import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
+import HttpClient from '@cdo/apps/util/HttpClient';
```

Comment:
Can you say a bit more about why you decided to change this?

Reply:
This is purely to make testing with Jest mocks easier. It seems like other places (student learning) in the code base that fetch from the BE use this HttpClient wrapper. It was much easier to mock this HttpClient than to try to mock the global fetch function.